pdns/ixfrdist.cc

```diff
@@ -610,109 +574,149 @@ void handleTCPRequest(int fd, boost::any&) {
     cfd = SAccept(fd, saddr);
     setBlocking(cfd);
```

Comment:
I don't think you need to set it blocking, it will be by default.

Reply:
I don't think it hurts right?